CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+# dbt_zendesk_source v0.10.1
+[PR #43](https://github.com/fivetran/dbt_zendesk_source/pull/43) introduces the following updates:
+
+## Feature Updates
+- Added the `internal_user_criteria` variable, which can be used to mark internal users whose `USER.role` may have changed from `agent` to `end-user` after they left your organization. This variable accepts SQL that may reference any non-custom field in `USER`, and it will be wrapped in a `case when` statement in the `stg_zendesk__user` model.
+  - Example usage:
+```yml
+# dbt_project.yml
+vars:
+  zendesk_source:
+    internal_user_criteria: "lower(email) like '%@fivetran.com' or external_id = 12345 or name in ('Garrett', 'Alfredo')" # can reference any non-custom field in USER
```

Comment:
Small suggestion, but when I ran this exact snippet locally, I got an error that the `=` operator may not be used when comparing STRING to INT. We should probably update this to be consistent that it will work if copy/pasting.

```suggestion
    internal_user_criteria: "lower(email) like '%@fivetran.com' or external_id = '12345' or name in ('Garrett', 'Alfredo')" # can reference any non-custom field in USER
```